ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -349,7 +349,7 @@ ORK_CLASS_AVAILABLE
  The text of the currently selected choice is displayed on screen. The text for
  each choice is spoken by VoiceOver when an image is highlighted.
  */
-@property (readonly, copy) NSArray *imageChoices;
+@property (readonly, strong) NSArray *imageChoices;
```

Comment:
This is intentionally copy, although it's irrelevant as API because it's a readonly property. Do not change.